ezissue/converter/manipulation.py

```diff
@@ -38,7 +38,7 @@ def format_description(string):
     """
     Adds the header and a final new line to the issue description string.
     """
-    return str('**Issue description:**\n---\n\n' + string + '\n'*2)
```

Comment:
Keep it in english for now. Translations will be added later. Btw there is a branch named `desenho-br` I think, that writes the issue titles and stuff in pt-br and is deployed on ezissue-br lib